# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: build/rust/env_logger/lib.rs
# Commit: 9493029c76b9
# Full Hash: 9493029c76b93861ca10dbfb29d4178773144416
# Author: zaggy1024 <Zaggy1024@gmail.com>
# Date: 2023-01-28 21:11:06
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Bug 1788119 - Part 4 - Update mp4parse-rust for AVIS support. r=kinetik,glandium,supply-chain-reviewers
#   
#   Depends on D156652
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156653
# ==============================================================================

diff -r 7d5cc211e3f5 -r 9493029c76b9 build/rust/env_logger/lib.rs
--- a/build/rust/env_logger/lib.rs	Fri Jan 27 19:54:29 2023 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,5 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-pub use env_logger::*;